tests/unit/test_elgamal.py

```diff
@@ -0,0 +1,63 @@
+from electionguard import ElGamalKeyPair
+from electionguard.elgamal import (
+    ElGamalPublicKey,
+    ElGamalSecretKey,
+    hashed_elgamal_encrypt,
+)
+from electionguard.group import ElementModQ
+from electionguard.serialize import PaddedDataSize, _add_padding, _remove_padding
```

Comment:
If we are using the `_add_padding` and `_remove_padding` like this. It should be denoted as a public function.